### ISSUE TITLE
The 'Pods-Toto' target has frameworks with conflicting names

### ISSUE THREAD

**Issue by vincedev** [CocoaPods/CocoaPods#4861]
I'm using CocoaPods 0.39.0 with a very simple Podfile, and I get the following error when running `pod install`.  

```
source 'https://github.com/CocoaPods/Specs.git'
workspace './Toto.xcworkspace'
xcodeproj './Toto.xcodeproj'

inhibit_all_warnings!
plugin "cocoapods-no-dev-schemes"
    
def import_ios_osx_but_tests_pods
    pod 'Fabric'
    pod 'Crashlytics'
end

target :'Toto' do
	platform :osx, '10.10'
    import_ios_osx_but_tests_pods

end

target :'Toto copy' do
    platform :osx, '10.10'
    import_ios_osx_but_tests_pods    
end

target :'Toto iOS' do
    platform :ios, '8.0'
    import_ios_osx_but_tests_pods    
end
```

The only way to get around the error is by commenting out the `target :'Toto iOS' do`.
How can I resolve this ?

**Comment by neonichu** [comment]
This is a bug in 0.39, it's fixed in the 1.0 betas. See #4575

**Comment by euroboy** [comment]
Got this error in 1.1.0.rc.2
